roles/kubernetes/master/tasks/kubeadm-migrate-certs.yml

```diff
@@ -1,20 +0,0 @@
----
```

Comment:
We want to retain non-kubeadm to kubeadm. Don't remove this.

Also, you'll need to handle the old cert path as src.